components/org.wso2.carbon.micro.integrator.core/src/main/java/org/wso2/carbon/micro/integrator/core/internal/ServiceComponent.java

```diff
@@ -395,7 +395,7 @@ private void initializeCarbon() {
             registerCarbonServlet(httpService, defaultHttpContext);
 
 
-            log.info("Repository       : " + axis2RepoLocation);
+            log.debug("Repository       : " + axis2RepoLocation);
```

Comment:
We can check if debug is enabled

Reply:
Fixed in https://github.com/wso2/micro-integrator/pull/65/commits/51edbc2cd480c5bf084bbea1edb41444a02c0b18